manifests/params.pp

```diff
@@ -1,6 +1,19 @@
 # archive parameters
 class archive::params {
   case $::osfamily {
+    'Windows': {
+      $path               = getvar('::archive_windir')
+      $owner              = 'S-1-5-32-544' # Adminstrators
+      $group              = 'S-1-5-18'     # SYSTEM
```

Comment:
Looool!
That's the "Administrators" users according to the Windows convention: https://support.microsoft.com/en-us/kb/243330
No wonder why it's been years I haven't touched a Windows box! :)